docs/backups-pitr.md

```diff
@@ -33,16 +47,18 @@ backup:
     timeBetweenUploads: 60
 ```
 
-!!! note
+For how to restore a database to a specific point in time, see [Restore the cluster with point-in-time recovery](backups-restore.md#restore-the-cluster-with-point-in-time-recovery).
+
+## Monitoring binary logs
+
+The point-in-time recovery Pod collects statistics metrics for binlogs. They provide insights into the success and failure rates of binlog operations, timeliness of processing and uploads and potential gaps or inconsistencies in binlog data.
```

Comment:
The point-in-time recovery Pod collects statistics metrics for binlogs -> pod is not collecting info and does not aggregate it, so after pod restart counters will be restarted and will grow until next restart - for example this one pxc_binlog_collector_success_total .